resources/automate.rb

```diff
@@ -106,14 +102,11 @@
     notifies :reconfigure, 'chef_ingredient[automate]', :immediately
   end
 
-  if new_resource.enterprise.is_a?(String)
-    new_resource.enterprise = [new_resource.enterprise]
-    new_resource.enterprise.each do |ent|
-      execute "create enterprise #{ent}" do
-        command "delivery-ctl create-enterprise #{ent} --ssh-pub-key-file=/etc/delivery/builder_key.pub > /etc/delivery/#{ent}.creds"
-        not_if "delivery-ctl list-enterprises --ssh-pub-key-file=/etc/delivery/builder_key.pub | grep -w #{ent}"
-        only_if 'delivery-ctl status'
-      end
+  (new_resource.enterprise.is_a?(String) ? [new_resource.enterprise] : new_resource.enterprise).each do |ent|
```

Comment:
> it is also preferred to call the property directly now

I've had mixed results with that (which I assume is my own misunderstanding of scope and context within certain methods), but that would simplify a lot!